packages/notebook/src/actions.tsx

```diff
@@ -1966,6 +2000,19 @@ namespace Private {
     activeCell: Cell | null;
   }
 
+  export function isNotebookRendered(notebook: Notebook): boolean {
+    if (notebook.cellsToRender.size !== 0) {
+      showDialog({
+        body: `Notebook is still rendering and has for now ${notebook.cellsToRender.size} remaining cells to render.
+
+Please wait for the complete rendering before invoking that action.`,
+        buttons: [Dialog.okButton({ label: 'Ok' })]
```

Comment:
_Ok_ needs translation too.

Reply:
I have added the default translator in the isNotebookRendered